addons/l10n_id/data/res.country.state.csv

```diff
@@ -0,0 +1,27 @@
+"id","country_id:id","code","name"
```

Comment:
If we have this in base, why do we need it here?  @jao-odoo

Reply:
@jco-odoo  it was removed in commit dbe454f1faa648dbbe4575e4af2987c4ca1d5510 but looks like bad force push 1ad947b305ec5b7452fb72430b6efa456e8adbad reintroduced this file.

Reply:
@rgo-odoo  @jco-odoo  Done